src/vl-core.js

```diff
@@ -1,266 +1,266 @@
 import '/node_modules/document-register-element/build/document-register-element.js';
 
-export const VlElement = (SuperClass) => {
+export const vlElement = (SuperClass) => {
```

Comment:
classes niet meer met een hoofdletter dan?

Reply:
breaking change overal dan?

Reply:
Classes wel, misins niet.

Reply:
dus moet op alle componenten doorgevoerd gaan worden eer ge er in een project van gebruik kunt maken? dan zou ik er nog eens goed over nadenken wat dit ons oplevert.